docs/portal/dashboard.md

```diff
@@ -1,9 +1,26 @@
-# Understanding the Dashboard
+# Dashboard
 
-The Dashboard appears when you first log in to the {{title}} portal.
+HDT Dashboard give you information about the traffic of your HDT transports.
```

Comment:
Please consider changing this text to: "The HDT Dashboard shows the traffic associated with your HDT transfers."

Reply:
OKay, will make the change.